backend/rates/gecko.go

```diff
@@ -54,9 +54,28 @@ var (
 		"eth-erc20-paxg":      "pax-gold",
 	}
 
+	// The keys are CoinGecok coin codes.
+	// The values are BitBoxApp coin units.
+	geckoCoinToUnit = map[string]string{
```

Comment:
feels inconsistent with the two new maps. I'd rename this to `fromGeckoCoin` and `geckoCoin` above to `toGeckoCoin`.

Reply:
But one is converting to the coin unit, the other from coin code, so they are not inverse.